pages/home.js

```diff
@@ -101,19 +105,35 @@ const Home = ({
     </PageSection>
 
     {featureFlags.release10 && (
-      <PageSection>
-        <Container>
-          <h2>{featuredBlogPostsTitle}</h2>
-        </Container>
-
-        <BlogPostList className="row">
-          {featuredBlogPosts.map(post => (
-            <BlogPostListItem key={post.slug} className="col-md-4">
-              <BlogPostCard post={post} />
-            </BlogPostListItem>
-          ))}
-        </BlogPostList>
-      </PageSection>
+      <Fragment>
+        <PageSection>
+          <h2>{bannersTitle}</h2>
+
+          <Banner
```

Comment:
Isn't this missing a "campaign is active" check?

Reply:
Yep you are right.

Reply:
Actually just tested with IsActive turned off and it still shows both on Home and Wie Sie helfen...